tldr/hybrid_extractor.py

```
@@ -3151,6 +3224,315 @@ def _extract_elixir_function(self, node, source: bytes, is_private: bool) -> Fun
+
+        for child in node.children:
+            if child.type == "name":
+                name = self._safe_decode(source[child.start_byte:child.end_byte])
+            elif child.type == "base_clause":
+                for c in child.children:
+                    if c.type == "name" or c.type == "qualified_name":
+                        bases.append(self._safe_decode(source[c.start_byte:c.end_byte]))
+            elif child.type == "class_interface_clause":
+                for c in child.children:
+                    if c.type == "name" or c.type == "qualified_name":
+                        bases.append(self._safe_decode(source[c.start_byte:c.end_byte]))
```

Ruff (0.15.7)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)